selenium/__mocks__/tintin.mock.js

```diff
@@ -73,4 +73,16 @@ module.exports = function krigerkattene(times) {
     .post('/search/?q=%28%22%2A%22%29%20AND%20%28phrase.titleSeries%3D%22Carlsen%20minicomics%22%29&fields%5B0%5D=pid&fields%5B1%5D=coverUrlFull&fields%5B2%5D=dcTitle&fields%5B3%5D=titleSeries&fields%5B4%5D=workType&limit=6&offset=0&sort=solr_numberInSeries_ascending')
     .times(times)
     .reply(200, {"statusCode":200,"data":[{"pid":["870970-basis:52000602"],"coverUrlFull":["//moreinfo.addi.dk/2.6/more_info_get.php?lokalid=52000602&attachment_type=forside_stor&bibliotek=870970&source_id=870970&key=16ae391a3e0a51f8fb4c"],"dcTitle":["Faraos cigarer"],"titleSeries":["Tintins oplevelser"],"workType":["book"]},{"pid":["870970-basis:29394393"],"coverUrlFull":["//moreinfo.addi.dk/2.6/more_info_get.php?lokalid=29394393&attachment_type=forside_stor&bibliotek=870970&source_id=870970&key=2544df996d2c85794ac7"],"dcTitle":["Den mystiske stjerne"],"titleSeries":["Tintins oplevelser"],"workType":["book"]},{"pid":["870970-basis:29693544"],"dcTitle":['"Enhjørningen"s hemmelighed'],"titleSeries":["Tintins oplevelser ; 11"],"workType":["book"]},{"pid":["870970-basis:29068623"],"coverUrlFull":["//moreinfo.addi.dk/2.6/more_info_get.php?lokalid=29068623&attachment_type=forside_stor&bibliotek=870970&source_id=870970&key=869df4bbd3358ca2a811"],"dcTitle":["De 7 krystalkugler"],"titleSeries":["Tintins oplevelser ; 13"],"workType":["book"]},{"pid":["870970-basis:26913063"],"dcTitle":['Tintin, reporteren fra "Petit vingtième", i Sovjetunionen'],"titleSeries":["Tintins oplevelser ; 1","Carlsen minicomics"],"workType":["book"]},{"pid":["870970-basis:26913098"],"dcTitle":["Tintin i Congo"],"titleSeries":["Tintins oplevelser ; 2","Carlsen minicomics"],"workType":["book"]}]});
+
+  nock('http://platform-i01:8080', {encodedQueryParams: true})
+    .post('/search/?q=%28%22%2A%22%29%20AND%20%28phrase.titleSeries%3D%22Carlsen%20minicomics%22%29&fields%5B0%5D=pid&fields%5B1%5D=coverUrlFull&fields%5B2%5D=dcTitle&fields%5B3%5D=titleSeries&fields%5B4%5D=workType&limit=18&offset=0&sort=solr_numberInSeries_ascending')
+    .times(times)
+    .reply(200, {
+      "statusCode": 200,
+      "data": [],
```

Comment:
Ligeledes her. Hvad testes der for?

Reply:
Samme som ovenfor. Et kald der ikke bliver mocket får nock til at smide en fejl i loggen, hvilket støjer i testresultatet men ikke har nogen effekt for om testen går i grøn/rød derfor er der indsat tomme responses for at reducere støj, men det har ingen indflydelse på resultatet af testen, hvad der svares eller om kaldet overhovedet mockes.